airbyte-cdk/java/airbyte-cdk/core/src/testFixtures/kotlin/io/airbyte/cdk/testutils/TestDatabase.kt

```diff
@@ -300,18 +298,21 @@ protected constructor(@JvmField val container: C) : AutoCloseable {
         }
 
         fun withSsl(sslMode: MutableMap<Any?, Any?>?): B {
-            return with(JdbcUtils.SSL_KEY, true)!!.with(JdbcUtils.SSL_MODE_KEY, sslMode)
+            return with(JdbcUtils.SSL_KEY, true).with(JdbcUtils.SSL_MODE_KEY, sslMode)
         }
 
         companion object {
-            val DEFAULT_CDC_REPLICATION_INITIAL_WAIT: Duration? = Duration.ofSeconds(5)
+            private val DEFAULT_CDC_REPLICATION_INITIAL_WAIT: Duration = Duration.ofSeconds(5)
```

Comment:
```
@JvmStatic protected val DEFXXX: Duration = XXX
   private set
```

Reply:
in java it looks like this  `getDEFAULT_CDC_REPLICATION_INITIAL_WAIT()` by using above convention. Not sure why. its part of nested ConfigBuilder class's companion not the TestDatabase

Reply:
what about @JvmField? That should get rid of the constructor, no?